src/components/tools/ToolCard.tsx

```
@@ -0,0 +1,85 @@
+            {tool.description}
+          </p>
+          <div className="flex flex-wrap gap-1.5">
+            {tool?.categories && tool.categories.slice(0, 3).map((category) => (
+              <Badge
+                key={category}
+                variant="secondary"
+                className="bg-[#141414] bg-opacity-80 text-xs text-gray-300 hover:bg-[#141414] hover:bg-opacity-100"
+              >
+                {category}
+              </Badge>
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)